client/app.js

```diff
@@ -92,4 +92,13 @@ porybox.service('io', function () {
   return io;
 });
 
+porybox.service('errorHandler', ['$mdToast', function ($mdToast) {
+  return reason => {
+    console.error(reason);
+    $mdToast.show(
+      $mdToast.simple().textContent('An unexpected error occured.').position('bottom right')
```

Comment:
I wonder whether it is confusing to have the toast at a different point on the screen.